.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+repos:
```

Comment:
What is the purpose of using pre-commit here? (Naive question)

Reply:
We use pre-commit to automatically scan (and fix when possible) some code lines contravening PEP-8's guidelines (like sorting imports, lines that are too long, etc.). I configured it so it would only apply to the agac_torch/ subfolder !

Reply:
Got it, thanks for the explanation!